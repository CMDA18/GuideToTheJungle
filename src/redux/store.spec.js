import createStore from './store'

describe('Store', () => {
  it('Creates a valid store', () => {
    expect(createStore()).toMatchSnapshot()
  })
})
