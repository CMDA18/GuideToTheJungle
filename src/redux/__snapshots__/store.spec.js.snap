// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store Creates a valid store 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
